### PR TITLE
Navbar and Header

### PR DESCRIPTION
**Pull request by peteryeungtt**
This is my first my portfolio project:
Add navbar and header using Flexbox

### CODE REVIEW

**Review by levy002 — COMMENTED** [review]
## Hello @peteryeungtt 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :x: :x: 

You have submitted a project, that doesn't have a professional Readme file. Kindly make sure to remove all comments instructions in your Readme file in order to keep it clean and professional.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by peteryeungtt** [comment]
@levy002 Hi I did the correct way how to create a readme file, this must be a mistake , please check again

**Comment by peteryeungtt** [comment]
@levy002 Can you please verify again

**Comment by levy002** [comment]
Hello @peteryeungtt. Please go ahead and submit again the PR link in the dashboard if you have removed all comments in your README as I suggested.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                            alt="Twitter Link" /></a>
+                </li>
+                <li>
+                    <a class="social-media-link" href="#"><img class="headline-link-img" src="images/medium.svg"
+                            alt="Medium Link" /></a>
+                </li>
+            </ul>
+        </div>
+
+        <img class="illustration" src="images/illustration.svg" alt="illustration" />
```
Comment (tugsem) [inline index.html]: - [ ] The background image should be imported from CSS background-image property for best practices. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  font-family: var(--font-poppins);
+  font-size: 1.6em;
+  font-style: normal;
+  line-height: 1.5em;
+}
+
+.headline-social-media-links {
+  display: flex;
+  align-items: center;
+}
```
Comment (tugsem) [inline styles.css]: - [ ] Kindly remove these default behavior of a ul list. Make sure the dots can't be seen to match the figma design.
image

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+.social-media-link {
+  margin-right: 1em;
+}
+
+.illustration {
+  position: absolute;
+  right: -280;
+  bottom: 0;
+  rotate: -17deg;
+}
```
Comment (tugsem) [inline styles.css]: - [ ] Please make sure your background image does not overlap the body and overflow-x is not possible. That would increase the user experience.
image

**Review by tugsem — CHANGES_REQUESTED** [review]
##  STATUS: REQUIRED CHANGES ♻️
Hello @peteryeungtt 👋 , 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights 🚀 
✔️linters are passing
✔️correct use of github flow
✔️professional documentation
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (divinecharlotte) [inline index.html]: - [ ] kindly as requested by the previous reviewer consider importing this background image from CSS background-image property for best practices. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info. and after that make sure that sure your background image does not overlap the body and overflow-x is not possible. That would increase the user experience.

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @peteryeungtt   , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 
- linters are passing 👍
- descriptive pr 👍 





 

## Required Changes ♻️



_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @peteryeungtt , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- Your commit messages are descriptive. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by peteryeungtt** [comment]
Merging created_navbar_and_header branch with main branch

**Pull request merged**